.github/workflows/cclib_pytest.yml

```diff
@@ -55,3 +55,47 @@ jobs:
           fail_ci_if_error: true
           verbose: false
         if: matrix.container == 'shivupa/cclib-ci:py37'
+      - name: Install pypa/build
+        run: |
+          python -m pip install build
+      - name: Build distribution packages (binary wheel and source tarball)
+        run: |
+          python -m build
+      - name: Store the distribution packages
+        uses: actions/upload-artifact@v3
+        with:
+          name: python-package-distributions
+          path: dist/
+        # TODO not sure why this doesn't work
+        # if: matrix.container == 'shivupa/cclib-ci:py310'
```

Comment:
Shouldn't this become `'shivupa/cclib-ci:py310-edge'` and a bit above this `'shivupa/cclib-ci:py37-edge'`

Reply:
No wonder codecov has been broken

Reply:
yeah it seems like we lost a bit of coverage since the list time codecov has been run but its good that it is fixed